src/app/notifications/components/notificationListEntry.tsx

```diff
@@ -32,6 +33,22 @@ export const NotificationListEntry: React.SFC<NotificationListEntryProps> = (pro
         history.push(path);
     };
 
+    const copyToClipboard = () => {
+        navigator.clipboard.writeText(message);
+    };
+
+    const CopyButton = (buttonProps: { isFlex: boolean }): JSX.Element => {
+        return (
+            <IconButton
+                iconProps={{ iconName: 'copy' }}
+                title={'copy'}
```

Comment:
We shall use the 'copy' text from resource keys, I believe there should already be one created.

Reply:
Thanks will change!